<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_received">
       <property name="text">
        <string>Received:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget_received">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_received">
        <attribute name="title">
         <string>received</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="textBrowser_received"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_received_hex">
        <attribute name="title">
         <string>Hex</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="textBrowser_received_hex"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_tosend">
       <property name="text">
        <string>To Send:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget_tosend">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_tosend">
        <attribute name="title">
         <string>to send</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit_tosend"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_tosend_hex">
        <attribute name="title">
         <string>Hex</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit_tosend_hex"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_port">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_open">
       <property name="text">
        <string>&amp;Open</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_send">
       <property name="text">
        <string>&amp;Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
